src/tribler-gui/tribler_gui/tribler_request_manager.py

```diff
@@ -91,7 +91,7 @@ def clear(self):
 
     def evict_timed_out_requests(self):
         t = time()
-        for req in list(self.requests_in_flight.items()):
+        for _, req in list(self.requests_in_flight.items()):
```

Comment:
Why not use ‘.values’?

Reply:
Because I'm old-school :eyeglasses: .

Seriously, just copy-pasted the stuff...